### ISSUE TITLE
[Feature] screenshot diff reporter

### ISSUE THREAD

**Issue by mousemke** [segmentio/niffy#10]
Post failed test, an html page could be generated and opened to show the diff failure.

**Comment by jessehattabaugh** [comment]
Agreed that this would be nice to have, but just having the screenshots in my project directory would be a start, since my diff-tool (Kaleidoscope) can diff images.
